### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by jksmall0631**

### CODE REVIEW

**Comment by jksmall0631** [comment]
Hey, got redux and router set up ish.

File: app/containers/MovieIndex-container.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { addMovies } from '../actions';
+import MovieIndex from '../components/MovieIndex';
```
Comment (stephanieea) [inline app/containers/MovieIndex-container.js:4]: Awesome!

File: app/containers/MovieIndex-container.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+const mapStateToProps = (state) => {
```
Comment (stephanieea) [inline app/containers/MovieIndex-container.js:6]: YES!

File: app/index.js [hunk 3]
```diff
@@ -1,15 +1,24 @@
 import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
-import App from './components/app';
-import MovieIndex from './components/movieIndex';
+import {createStore} from 'redux';
```
Comment (stephanieea) [inline app/index.js:3]: BOOOOOMMMMMM!!!

File: app/index.js [hunk 4]
```diff
@@ -1,15 +1,24 @@
...
+import App from './containers/App-container';
+import MovieIndex from './containers/MovieIndex-container';
 import { Router, Route, IndexRoute, browserHistory } from 'react-router';
+import allReducers from './reducers/index';
+import {Provider} from 'react-redux';
+
+const devTools = window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__();
```
Comment (stephanieea) [inline app/index.js:10]: YAY Dev Tools!

File: app/reducers/index.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import {combineReducers} from 'redux';
```
Comment (stephanieea) [inline app/reducers/index.js:1]: DOPE!

File: app/reducers/reducer-app.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+const appReducer = (state = [], action) => {
```
Comment (stephanieea) [inline app/reducers/reducer-app.js:1]: SWEET!!

File: app/index.js (hunk 4 above)
Comment (devinmarieb) [reply app/index.js:10]: hellssss ya boi

**Review by stephanieea — APPROVED** [review]

**Pull request merged**